css/styles.css

```diff
@@ -84,6 +84,39 @@ form {
   margin-top: 25px;
 }
 
+.hidden {
```

Comment:
I would move this to the bottom of the css document, but above the media queries. That is to ensure that no CSS after this resets the display property of any hidden elements.